sources/packages/backend/libs/services/src/constants/error-code.constants.ts

```diff
@@ -32,3 +32,9 @@ export const UNEXPECTED_ERROR_DOWNLOADING_FILE =
 export const ECE_DISBURSEMENT_DATA_NOT_VALID =
   "ECE_DISBURSEMENT_DATA_NOT_VALID";
 export const FILE_PARSING_ERROR = "FILE_PARSING_ERROR";
+// Error code used when file virus scanning failed for some reason.
```

Comment:
Minor: Error codes used.....

Reply:
This file has been reverted. The comment is also updated.